tests/other/metadce/mem_no_argv_O3_STANDALONE_WASM_flto.funcs

```diff
@@ -1,4 +1,3 @@
 $_start
 $main
 $malloc
-$sbrk
```

Comment:
Interesting, is this difference due to changing LTO from `-O3` to `-O2`?

Maybe we should investigate the code size impact of this change. However it seems like it it didn't break any metadce code size checks then the change is small.

Reply:
It seems this is because we were previously defaulting to passing `-Wl,-lto-O0` if `--llvm-lto` was not passed to emscripten at all.  (This doesn't was passing `-flto` but not `--llvm-lto`)

Now we let it default which is doing -lto-O2.

Reply:
Oh right, makes sense.